CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Change Log
 
+## 2.4
+
+This is a release which focuses on performance and bug-fixes.
+
+### Performance
+
+- Improve swapchain latency and frame pacing by up to one frame.
+- Optimize lookup of format info.
+- Avoid potential pipeline compilation stutter in certain scenarios.
+- Rewrite how we handle image layouts for color and depth-stencil targets.
+  Allows us to remove a lot of dumb
+  barriers giving significant GPU-bound performance improvements.
+  ~15%-20% GPU bound uplift in Horizon Zero Dawn,
+  ~10% in Death Stranding,
+  and 5%-10% improvements in many other titles.
+
+### Features
+
+- Enable tiled resources tier 3 if supported (3D textures).
```

Comment:
should probably say "sparse 3D texttures" or just nothing at all, otherwise the Phoronix headline will read "vkd3d-proton 2.4 released with initial support for 3D textures".